Formula/purescript.rb

```diff
@@ -1,7 +1,7 @@
 class Purescript < Formula
   desc "Strongly typed programming language that compiles to JavaScript"
   homepage "https://www.purescript.org/"
-  url "https://hackage.haskell.org/package/purescript-0.14.2/purescript-0.14.2.tar.gz"
+  url "https://hackage.haskell.org/package/purescript-0.14.3/purescript-0.14.3.tar.gz"
   sha256 "b538dc52b30712d6efd211da3bffa72f77a1e23b49973b017c69ec100623f389"
```

Comment:
Perhaps you forgot to update the checksum?:
```
Error: SHA256 mismatch
Expected: b538dc52b30712d6efd211da3bffa72f77a1e23b49973b017c69ec100623f389
  Actual: 2a82532d416d93d117ab942ab0b618642d788bf0dae09fc045bc1cc88ba71f3c
```

Reply:
thx, updated